ci_scripts/assert_news.py

```diff
@@ -50,8 +50,8 @@ def _verify_news_file_content(self):
             raise FileNotFoundError(self._news_file_path)
         with open(self._news_file_path, 'r') as fh:
             file_content = fh.read()
-        if not file_content:
-            raise ValueError(f'Empty news file `{self._news_file_path}`')
+        if not file_content or len(file_content.strip()) == 0:
+            raise ValueError(f'Empty news file `{self.news_file_path}`')
```

Comment:
Looks like there are other `_news_file_paths` that need to be updated to `news_file_path`?